### PR TITLE
Some plotting bug fixes

### PR DESCRIPTION
**Pull request by hoetzsch**
- fix ylim for 1D equiprobable-binned plot function (full ylim range of plot is now used by using axvspan instead of Rectangles)
- fix colormap in plot vs colorbar alpha mismatch

### CODE REVIEW

File: GOFevaluation/utils.py [hunk 1]
```diff
@@ -390,10 +392,11 @@ def plot_equiprobable_histogram(data_sample, bin_edges, order=None,
...
         for low, high in zip(bin_edges[:-1], bin_edges[1:]):
-            rec = Rectangle((low, 0), high - low, 1,
-                            facecolor=cmap(norm(ns[i])),
-                            **kwargs)
-            ax.add_patch(rec)
+            ax.axvspan(low,
+                       high,
+                       facecolor=cmap(norm(ns[i])),
+                       **kwargs,
+                       )
```
Comment (hammannr) [inline GOFevaluation/utils.py:399]: just fixing an old bug here: We should put `edgecolor = kwargs.pop('edgecolor', 'k')` further up in the code and then pass this also to the axvspan (as we do it in the 2d case with the rectangle).

**Review by hammannr — COMMENTED** [review]
This looks very nice! Just one small comment on fixing an old bug that I just noticed now when testing the vspan, then this PR is good to go!

File: GOFevaluation/utils.py (hunk 1 above)
Comment (hoetzsch) [reply GOFevaluation/utils.py]: done!

**Review by hammannr — APPROVED** [review]
Awesome!! Many thanks for spotting and fixing this bug so quickly!! ⚡

**Pull request merged**